src/components/Top-Bar.vue

```diff
@@ -67,6 +61,45 @@ export default {
 }
 </script>
 
-<style scoped="true">
+<style scoped lang="scss">
```

Comment:
We dropped scss. do we need it again?

Reply:
this is because the docker build is failing I guess (havn't had a look yet)

Reply:
Of course, we could also get rid of make completely and use an npm-baseimage for the pipeline. This would allow us to use sass/scss again, given that we are starting to develop our own base components.